### PR TITLE
Renaming button + correct 2 action type + change icon name

### PR DESCRIPTION
**Pull request by alkabal**

### CODE REVIEW

File: psng/python/tool_measurement.py [hunk 1]
```diff
@@ -164,8 +163,17 @@ def on_spbtn_block_height_value_changed(self, gtkspinbutton, data=None):
...
+        # Start psng_probe_table.ngc
+        if self.ocode("o<psng_probe_table> call") == -1:
+            return
+        a = self.probed_position_with_offsets()
+        self.add_history(gtkbutton.get_tooltip_text(), "Z", z=a[2])
+        self.set_zerro("Z", 0, 0, a[2])
+        
     # Down probe to tool setter for measuring it vs table probing result
-    def on_btn_probe_tool_setter_released(self, gtkbutton, data=None):
+    def on_btn_probe_setter_released(self, gtkbutton, data=None):
```
Comment (kiall) [inline psng/python/tool_measurement.py:176]: Why rename this? `probe_tool_setter` is IMO clearer, and now there is a mix of both `probe_tool_setter` and `probe_setter` in the code.

File: psng/python/tool_measurement.py [hunk 2]
```diff
@@ -182,17 +190,14 @@ def on_btn_probe_workpiece_released(self, gtkbutton, data=None):
...
-
     # Probe tool Diameter
-    def on_tool_dia_released(self, gtkbutton, data=None):
+    def on_btn_tool_dia_released(self, gtkbutton, data=None):
+        tooltable = self.inifile.find("EMCIO", "TOOL_TABLE")
+        if not tooltable:
+            self.error_dialog(
+                "Tool Diameter Measurement Error",
+                secondary="Did not find a toolfile file in [EMCIO] TOOL_TABLE",
+            )
```
Comment (kiall) [inline psng/python/tool_measurement.py:200]: Why was this added? Does the method not work if a tool table file is not specified? As I understand it, this will work just fine without the tool table (I've not tested this understanding!)

File: psng/python/base.py [hunk 3]
```diff
@@ -335,7 +335,7 @@ def set_zerro(self, s="XYZ", x=0.0, y=0.0, z=0.0):
             if "Z" in s:
                 tmpz = tmpz - z + self.halcomp["ps_offs_z"]
                 c += " Z%s" % tmpz
-            self.gcode(c)
+            self.gcode(c)                                 # Is there a real reason for not use the same system if self.gcode(c) == -1: for coding style ?
```
Comment (kiall) [inline psng/python/base.py:338]: This functional call probably should use `if self.gcode(c) == -1:`, returning -1 from this method on failure and 0 on success, just as the `def gcode` method does.

**Review by kiall — COMMENTED** [review]
Re the rename of `down.png` - I don't think we should do this, for 2 reasons - 1) The icon is showing "probe down" - so the current name IMO makes sense. 2) Ideally we'd be able to stop distributing the images at all and just reuse the files included with LinuxCNC - https://github.com/LinuxCNC/linuxcnc/blob/2.8/share/qtvcp/images/probe_icons/down.png

**Comment by alkabal** [comment]
Ok 

I will revert the change name unwanted soon


The whole tool diameter method is not correct imo this need to be change like my fork and need to be saved in the tooltable so the tooltable is required but you can keep like it is if you think is better

Br

**Comment by kiall** [comment]
> The whole tool diameter method is not correct imo this need to be change like my fork and need to be saved in the tooltable so the tooltable is required but you can keep like it is if you think is better

If the tooltable is required for a later change, adding that code should IMO only happen once that other change is made - right now, it doesn't add any value and means whatever that later change is, is spread over multiple PRs instead of 1 PR being 1 change.
